src/person.rb

```diff
@@ -1,9 +1,12 @@
-class Person
+require_relative './decorate'
+
+class Person < Decorate
```

Comment:
- [ ] The Person class should inherit from Nameable, not Decorator. Kindly fix that